src/components/RunTracker/RunTracker.js

```diff
@@ -0,0 +1,70 @@
+import React, { useState } from "react";
+import Run from "../../utils/Run";
+import RunRecordsContainer from "../RunRecordsContainer/RunRecordsContainer";
+import "../../styles/RunTracker.css";
+import Button from "react-bootstrap/Button";
+import NewRunRecordForm from "../NewRunRecordForm/NewRunRecordForm";
+import { v4 as uuidv4 } from "uuid";
+import Header from "../Header/Header";
+import Statistics from "../Statistics/Statistics";
+import RunRecords from "../RunRecords/RunRecords";
+
+const RunTracker = () => {
+    const initial_run_records = createInitialRunRecords();
+    const [runRecords, setRunRecords] = useState(initial_run_records);
+    const [addNewRecordClicked, setAddNewRecordClicked] = useState(false);
```

Comment:
it is a name of a function and not a boolean variable
maybe : isAllowAddNewRecord, canAddNewRecord , isNewRecordAddable... 
or isAddNewRecordClicked 

depend on the scenario.

Reply:
or isEditMode